includes/class.llms.post-types.php

```diff
@@ -1456,34 +1444,62 @@ public static function register_taxonomies() {
 	}
 
 	/**
-	 * Add submenu items for earned engagments post types (llms_my_certificate,llms_my_achievement).
+	 * Modify submenu item links for llm_my_certificate and llms_my_achievement.
+	 *
+	 * By default the post types submenu items link to the post type list table,
+	 * we want them to point to the "Add New" screen.
 	 *
 	 * @since [version]
 	 *
 	 * @return void
 	 */
-	public static function add_earned_egnagements_submenu_links() {
+	public static function modify_earned_engagements_submenu_links() {
```

Comment:
I think this is better suited in this file: https://github.com/gocodebox/lifterlms/blob/trunk/includes/admin/class.llms.admin.menus.php